bundles/io/org.openhab.io.rest/src/main/java/org/openhab/io/rest/internal/listeners/ResourceStateChangeListener.java

```diff
@@ -54,6 +54,7 @@
 	
 	static ScheduledFuture<?> executorFuture;
 	
+	protected Item lastChange;
```

Comment:
Yes, In openHAB are ~5 Items which are triggered by an addon this causes to ~5 new updates at the same time.